editions/tw5.com/tiddlers/wikitext/Macro Definitions in WikiText.tid

```diff
@@ -19,7 +19,17 @@ Alternatively, the entire definition can be presented on a single line, without
 
 A more formal [[presentation|Macro Definition Syntax]] of this syntax is also available.
 
-!! Placeholders
+!!Accessing variables and parameters
```

Comment:
`!!<space>Accessing ...

Reply:
Thanks. Forgive my ignorance of github but does this mean you've already made that change or is it only being pointed out and I'm supposed to make the modification somehow?

Reply:
I just wanted to point it out. You can change it

Reply:
just edit the file and commit it. It will be automatically added to this PR.